src/stack.js

```diff
@@ -11,14 +11,14 @@ export default class Stack {
   }
 
   pop = () => {
-    if (this.size == 0) return null
+    if (this.size === 0) return null
```

Comment:
I recommend avoiding inline control flow, and replication of code (use `empty()` here).